### ISSUE TITLE
Crash at login

### ISSUE THREAD

**Issue by arnaudsm** [jellyfin/jellyfin#4318]
**Describe the bug**
The web client crashes when login in, with an infinite load animation. [Screenshot](https://i.imgur.com/DxaEdly.png).
But when I reopen the website in a new tab, the login was successful and my library shows flawlessly.

My instance is 1 year old, the issue only appeared with 10.6.x

**System :**
 - Virtualization: Docker
 - Clients: Web
 - Browser: Chrome 85 (no extensions)
 - Jellyfin Version: 10.6.3
 - Installed Plugins: None
 - Reverse Proxy: nginx

**To Reproduce**
1. Login 
2. See freeze
3. Open website in new tab
4. Everything works again

**Expected behavior**
Login should not crash the page, but display the library.

**Logs**
Server logs 
```
[2020-10-12 20:02:02.252 +00:00] [INF] [62] Jellyfin.Server.Implementations.Users.UserManager: Authentication request for "xxxxxxxxx" has succeeded.
[2020-10-12 20:02:02.252 +00:00] [INF] [62] Emby.Server.Implementations.Session.SessionManager: Creating new access token for user xxxxxxxxxxxx-xxxxx-xxxx-xxxx-xxxxxxxxxxxxxx
[2020-10-12 20:02:02.291 +00:00] [INF] [54] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS "::ffff:xxx.xx.x.x" request
[2020-10-12 20:02:02.302 +00:00] [ERR] [77] Emby.Server.Implementations.HttpServer.HttpListenerHost: Error processing request. URL: "http://xxxxxxx/jellyfin/DisplayPreferences/usersettings?userId=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx&client=emby"
System.Text.Json.JsonException: 'h' is invalid after a value. Expected either ',', '}', or ']'. Path: $.CustomPrefs.tvhome | LineNumber: 0 | BytePositionInLine: 474.
 ---> System.Text.Json.JsonReaderException: 'h' is invalid after a value. Expected either ',', '}', or ']'. LineNumber: 0 | BytePositionInLine: 474.
   at System.Text.Json.ThrowHelper.ThrowJsonReaderException(Utf8JsonReader& json, ExceptionResource resource, Byte nextByte, ReadOnlySpan`1 bytes)
   at System.Text.Json.Utf8JsonReader.ConsumeNextToken(Byte marker)
   at System.Text.Json.Utf8JsonReader.ConsumeNextTokenOrRollback(Byte marker)
   at System.Text.Json.Utf8JsonReader.ReadSingleSegment()
   at System.Text.Json.Utf8JsonReader.Read()
   at System.Text.Json.JsonSerializer.ReadCore(JsonSerializerOptions options, Utf8JsonReader& reader, ReadStack& readStack)
   --- End of inner exception stack trace ---
   at System.Text.Json.ThrowHelper.ReThrowWithPath(ReadStack& readStack, JsonReaderException ex)
   at System.Text.Json.JsonSerializer.ReadCore(JsonSerializerOptions options, Utf8JsonReader& reader, ReadStack& readStack)
   at System.Text.Json.JsonSerializer.ReadCore(Type returnType, JsonSerializerOptions options, Utf8JsonReader& reader)
   at System.Text.Json.JsonSerializer.ParseCore(ReadOnlySpan`1 utf8Json, Type returnType, JsonSerializerOptions options)
   at System.Text.Json.JsonSerializer.Deserialize[TValue](ReadOnlySpan`1 utf8Json, JsonSerializerOptions options)
   at Emby.Server.Implementations.Data.SqliteDisplayPreferencesRepository.GetDisplayPreferences(String displayPreferencesId, Guid userId, String client)
   at Emby.Server.Implementations.Data.SqliteDisplayPreferencesRepository.GetDisplayPreferences(String displayPreferencesId, String userId, String client)
   at MediaBrowser.Api.DisplayPreferencesService.Get(GetDisplayPreferences request)
   at Emby.Server.Implementations.Services.ServiceExecGeneral.Execute(Type serviceType, IRequest request, Object instance, Object requestDto, String requestName)
   at Emby.Server.Implementations.Services.ServiceController.Execute(HttpListenerHost httpHost, Object requestDto, IRequest req)
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost httpHost, IRequest httpReq, HttpResponse httpRes, ILogger logger, CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
```

Chrome JS console 
```
Error opening web socket: Error: Cannot open web socket without access token.
bundle.js?v=12:266 returning instance from getOrAddApiClient
bundle.js?v=12:266 Requesting url without automatic networking: https://xxxxxxx/jellyfin/Users/authenticatebyname
bundle.js?v=12:266 Requesting url without automatic networking: https://xxxxxxx/jellyfin/Sessions/Capabilities/Full
bundle.js?v=12:266 calling apiClient.ensureWebSocket
bundle.js?v=12:266 opening web socket with url: wss://xxxxxxx/jellyfin/socket?api_key=xxxxxxxxxxx&deviceId=xxxxxxxxxxxxxxxxxxxxxxxx
bundle.js?v=12:266 returning instance from getOrAddApiClient
bundle.js?v=12:266 Requesting url without automatic networking: https://xxxxxxx/jellyfin/DisplayPreferences/usersettings?userId=xxxxxxxxxxxxxxxxxxx&client=emby
bundle.js?v=12:266 web socket connection opened
bundle.js?v=12:266 GET https://xxxxxxx/jellyfin/DisplayPreferences/usersettings?userId=xxxxxxxxxxxxxxxxxxx&client=emby 500
index.html#!/login.html?serverid=xxxxxxxxxxx:1 Uncaught (in promise) Response {
    body: (...)
    bodyUsed: false
    headers: Headers {}
    ok: false
    redirected: false
    status: 500
    statusText: ""
    type: "basic"
    url: "https://xxxxx/jellyfin/DisplayPreferences/usersettings?userId=xxxxxxxxxxxxx&client=emby"
}
Sending web socket message: KeepAlive
```

**Comment by BaronGreenback** [comment]
Possibly https://github.com/jellyfin/jellyfin/issues/4310

**Comment by arnaudsm** [comment]
Solved by upgrading to 10.7.0
